### ISSUE TITLE
Question: direction detection? ("radar")

### ISSUE THREAD

**Issue by Sjoerd82** [roflcoopter/viseron#374]
Hi, I'm in the planning stage of adding Home Assistant to my house, and am planning for two camera's, both are near doors (front door, back door).

Is it feasable (and what is involved?) to have Viseron detect movement towards or away from the camera? And if so, could it fire an event that an object is approaching/moving away from door/zone? Goal here is to switch lights on/off, or do something else based on a person coming/going.

Basically act in many ways similar to radar.

**Comment by roflcoopter** [comment]
Hi!

You could setup several zones and then act depending on ther activation order.

I use Viseron to activate my lights around the door using the object_detected entities in Home Assistant, but direction is a bit more tricky.

**Comment by Sjoerd82** [comment]
Thanks! Yes, I can see this could potentially work 👍
